.buildkite/x-pack/pipeline.xpack.dockerlogbeat.yml

```diff
@@ -2,15 +2,20 @@
 name: "beats-xpack-dockerlogbeat"
 
 env:
-  AWS_ARM_INSTANCE_TYPE: "t4g.xlarge"
   BEATS_PROJECT_NAME: "x-pack/dockerlogbeat"
```

Comment:
I guess we can search and replace this in the pipeline, and this way we don't have `BEATS_PROJECT_NAME` anymore